schema/subnetGroupsSchema.json

```diff
@@ -13,18 +13,28 @@
       "subnetExplorerUriId": {
         "type": "string"
       },
-      "mainnetSubnetIds": {
-        "type": "array",
-        "items": {
-          "type": "string",
-          "pattern": "^[a-zA-Z0-9]+$"
-        }
+      "mainnetSubnetId": {
+        "type": "string",
+        "pattern": "^[a-zA-Z0-9]+$"
+      },
+      "testnetSubnetId": {
+        "type": "string",
+        "pattern": "^[a-zA-Z0-9]+$"
       },
-      "testnetSubnetIds": {
+      "chainGroups": {
         "type": "array",
         "items": {
-          "type": "string",
-          "pattern": "^[a-zA-Z0-9]+$"
+          "type": "object",
+          "properties": {
+            "mainnet": {
+              "type": "number",
+              "pattern": "^[1-9]d*$"
```

Comment:
Does this regex work? I think it should be `^[1-9]\d*$` and I'm not sure regex does anything to number types. Might want to use `minimum` instead.